src/bot_quake.py

```diff
@@ -146,15 +158,16 @@ async def handle_message(update, context):
 
 
 def main() -> None:
-    TOKEN = os.environ["TELEGRAM_BOT"]
+    """ Funzione principale del bot"""
+    token_bot = os.environ["TELEGRAM_BOT"]
```

Comment:
dato che il token non cambia mai ti consiglio di definire il nome della variabile in maiuscolo, come fatto prima.

Reply:
Il linter mi consigliava di usare lo snake_case altrimenti me lo scegnalava come errore. Che faccio cambio?

Reply:
Ok nessun problema, ignora il mio consiglio.
Probabilmente te lo segnala come errore perchè il linter non comprende che quella è una variabile in sola lettura.

Reply:
Ok, grazie comunque!